src/vtquery.cpp

```diff
@@ -295,17 +292,20 @@ struct Worker : Nan::AsyncWorker {
     void HandleOKCallback() override {
         Nan::HandleScope scope;
 
-        // std::string result("longitude: " + lng);
-        const auto argc = 2u;
+        results_to_json_string(result_string_, results_);
+
+        auto const argc = 2u;
```

Comment:
Add a `constexpr` to this perhaps? Also do not use `auto` here and specify directly your type.

Reply:
I honestly have no idea what this line is - [got it from node-cpp-skel](https://github.com/mapbox/node-cpp-skel/blob/1b8778737376e9ff786bf23c3a71b43349cb1fca/src/standalone_async/hello_async.cpp#L100)! cc @GretaCB @springmeyer

Reply:
I also have no idea why I added this line. I took a look at the history and the example links provided, nothin. I googled the line and found it in [one of @daniel-j-h 's repos](https://github.com/mapbox/node-or-tools/blob/master/src/vrp_worker.h#L225), which I mightve been working from.